bundles/net.tourbook.device.garmin/src/net/tourbook/device/garmin/GarminSAXHandler.java

```diff
@@ -111,69 +109,71 @@
-   private ArrayList<TimeData>       _allTimeData = new ArrayList<>();
-
-   private TimeData                  _timeData;
-
-   private int                       _dataVersion = -1;
-   private int                       _lapCounter;
-   private int                       _trackPointCounter;
-
-   private boolean                   _isSetLapMarker;
-   private boolean                   _isSetLapStartTime;
-   private ArrayList<Long>           _allLapStart = new ArrayList<>();
-
-   private DateTime                  _currentStartTime;
-   private long                      _currentTime;
-   private String                    _activitySport;
-   private int                       _tourCalories;
-   private int                       _lapCalories;
-   private boolean                   _isDistanceFromSensor;
-   private boolean                   _isFromStrideSensor;
-
-   private StringBuilder             _characters  = new StringBuilder();
-
-   private Sport                     _sport;
-   private String                    _tourNotes;
-   private String                    _tourTitle;
+   private boolean                 _importState_IsIgnoreSpeedValues;
+
+   private boolean                 _isInActivity;
+   private boolean                 _isInCourse;
+   private boolean                 _isInLap;
+
+   private boolean                 _isInAltitude;
+   private boolean                 _isInCalories;
+   private boolean                 _isInCadence;
+   private boolean                 _isInDistance;
+   private boolean                 _isInHeartRate;
+   private boolean                 _isInHeartRateValue;
+   private boolean                 _isInLatitude;
+   private boolean                 _isInLongitude;
+   private boolean                 _isInName;
+   private boolean                 _isInNotes;
+   private boolean                 _isInNs2Speed;
+   private boolean                 _isInNs2Watts;
+   private boolean                 _isInNs3Speed;
+   private boolean                 _isInNs3Watts;
+   private boolean                 _isInRunCadence;
+   private boolean                 _isInSensorState;
+   private boolean                 _isInTime;
+   private boolean                 _isInTrackpoint;
+
+   private boolean                 _isInCreator;
+   private boolean                 _isInCreatorName;
+   private boolean                 _isInCreatorVersionMajor;
+   private boolean                 _isInCreatorVersionMinor;
+
+   private HashMap<Long, TourData> _alreadyImportedTours;
+   private HashMap<Long, TourData> _newlyImportedTours;
+   private TourbookDevice          _device;
+   private String                  _importFilePath;
+   private boolean                 _isImported;
+
+   private final List<Long>        _pausedTime_Start  = new ArrayList<>();
+   private List<Long>              _pausedTime_End    = new ArrayList<>();
+   private int                     _currentPauseIndex = -1;
+   private boolean                 _isPreviousLapAPause;
```

Comment:
`_isPreviousLapAPause` Typo?

Reply:
No that looks good to me. This variable is used to indicate if a previously found lap was a pause.

Reply:
After reading you comment, this field name is OK it just looked strange when I first saw it.